core/src/test/scala/cell/base.scala

```diff
@@ -1553,9 +1553,9 @@ class BaseSuite extends FunSuite {
   }
 
   test("if exception-throwing tasks should still run quiescent handlers") {
-    val intMaxLattice = new Lattice[Int] {
-      def join(current: Int, next: Int) = math.max(current, next)
-      def empty = 0
+    val intMaxLattice = new MonotonicUpdater[Int] {
+      override def lteq(x: Int, y: Int): Boolean = x <= y
+      def bottom = 0
```

Comment:
As before, this should remain a lattice, and the implicit conversion should turn it into an updater.